README.md

```diff
@@ -32,14 +32,15 @@ cargo run --example basic
 ```
 It will do following things:
 1. Collect all environments(first-level directory) under `basic-case`.
-2. Run testcases(`.sql` files) under environment one after one.
-   1. Write temporary result to `{testcase}.output`
-   2. Compare `{testcase}.output` with `{testcase}.result` using `diff`
+2. Run tests(`.sql` files) under environment one after one.
+   1. Before execution it will read `{testcase}.result`(create one if not exists) to `Vec` as compare base.
+   2. During execution it will collect query response and write to `{testcase}.result`
+   3. After execution it will diff content of `Vec` with `{testcase}.result`, **PASS** when they are the same otherwise **FAIL**.
```

Comment:
```suggestion
   3. After execution it will compare the generated `{testcase}.result` with previous one, **PASS** when they are the same, and **FAIL** otherwise.
```